tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestSupplementalInfo.java

```diff
@@ -2032,6 +2032,10 @@ public void TestMetazones() {
                 ImmutableSet.of("America/Montreal", "Asia/Barnaul", "Asia/Tomsk", "Europe/Kirov");
         for (String timezoneRaw : TimeZone.getAvailableIDs()) {
             String timezone = TimeZone.getCanonicalID(timezoneRaw);
+            if (timezone.equals("Etc/Unknown")) {
+                System.err.println("CLDR-17949: Skipping " + timezone + " for raw " + timezoneRaw);
```

Comment:
Shouldn't we do this with logKnownIssue? But approving anyway to get things unblocked.

Reply:
Ideally, yes, but was just trying to get unblocked